src/Numerics/DGMethods/DGModel.jl

```diff
@@ -818,8 +831,31 @@ function reverse_indefinite_stack_integral!(
     wait(device, event)
 end
 
-# TODO: Move to BalanceLaws
-function nodal_update_auxiliary_state!(
+# By default, we call update_auxiliary_state!, given
+# nodal_update_auxiliary_state!, defined for the
+# particular balance_law:
+function update_auxiliary_state!(
+    dg::DGModel,
+    balance_law::BalanceLaw,
+    state_prognostic,
+    t,
+    elems,
+)
+    update_auxiliary_state!(
+        nodal_update_auxiliary_state!,
+        dg,
+        balance_law,
+        state_prognostic,
+        t,
+        elems,
+    )
+end
+
+# Should we provide a fallback implementation here?
+# Maybe better to throw a method error?
+function nodal_update_auxiliary_state!(balance_law, state, aux, t) end
```

Comment:
@simonbyrne we could remove this definition / throw a method error so that users must define `update_auxiliary_state!` or `nodal_update_auxiliary_state!` (and same for the init methods) Thoughts?